### PR TITLE
added alt text for a11y

### LINKED ISSUE #6294: [Accessibility] Agenda: Voice over does not convey the name of the image under Agenda screen.
### Target Platforms

iOS

### SDK Version

App Version: Version 1.0 (2.3.1-beta.20210603.1)

### Application Name

Visualizer

### Problem Description

[33767249](https://microsoft.visualstudio.com/OS/_workitems/edit/33767249)

Voice over does not convey the name of the image. It is announcing only as 'Image'. Voice over should convey the name of the image or make it non interactive so that focus should not land on it.

### Screenshots

_No response_

### Card JSON

[Agenda.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.2/Scenarios/Agenda.json)

### Sample Code Language

_No response_

### Sample Code

_No response_

### PR DESCRIPTION
**Pull request by jwoo-msft**
# Related Issue
Fixed #6294 

# Description
Updated sample with alt text for image
fixed broken urls to image of food order sample cards

# Sample Card

# How Verified

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/7595)

### CODE REVIEW

File: samples/v1.0/Scenarios/FoodOrder.json [hunk 1]
```diff
@@ -21,17 +21,17 @@
 			"images": [
 				{
 					"type": "Image",
-					"url": "https://contososcubademo.azurewebsites.net/assets/steak.jpg",
```
Comment (almedina-ms) [inline samples/v1.0/Scenarios/FoodOrder.json:24]: is this change necessary for this PR?

Comment (jwoo-msft) [reply samples/v1.0/Scenarios/FoodOrder.json:24]: this isn't needed to fix the 6294, but i wanted put sample updates together.

**Review by almedina-ms — APPROVED** [review]
